habitat/datasets/rearrange/rearrange_generator.py

```diff
@@ -866,6 +881,10 @@ def get_config_defaults() -> CN:
     #  }
     _C.markers = []
 
+    # If true, the episode will contain a single target. In addition, all AO will have their default (non-sampled) joint articulation EXCEPT
+    # for the AO receptacle containing the target (if any)
+    _C.unique_target = True
```

Comment:
Thanks for adding the comments. 
Seems like the edge case, modified behavior should not be default.
```suggestion
    #TODO: consider refactoring the generator to support this option more generically.
    _C.unique_target = False
```